### ISSUE TITLE
Metadata stripping

### ISSUE THREAD

**Issue by msurguy** [hicetnunc2000/hicetnunc#39]
When uploading files, there might be metadata embedded in the file that could be potentially a personal security problem - such as GPS coordinates in photos, EXIF metadata, etc. It would be great to show a prompt before uploading to let the user decide if they're ok with that metadata being included or if they want to clear metadata embedded in the file.

**Comment by andrevenancio** [comment]
Most users would not know how to do that I will imagine? But okay. tagging it as a feature request

**Comment by oldominion** [comment]
Yeah, actually this is a high priority concern. It could open the door to physical attacks on artists who's work sells for a lot of money.

**Comment by andrevenancio** [comment]
PR's are always welcome. We donate our own time to improve the platform. which is what makes open source great.
Here's some resources for your to start a PR with this feature.

https://stackoverflow.com/questions/27638402/strip-exif-data-from-image

**Comment by oldominion** [comment]
I'm aware. I would if I could, sadly I'm only proficient in .NET. To clarify, I know you guys are very busy right now, and my comment wasn't a demand for action, more to push this issue toward a (critical security) bug tag instead of just a feature request.

**Comment by daveyang** [comment]
Perhaps simply provide (on the upload page or the FAQ) a list of apps or services that remove metadata?

For example, I use: https://imageoptim.com/
